src/datapump/sync/sync.py

```diff
@@ -723,6 +723,122 @@ def get_latest_release(self) -> Tuple[str, List[str]]:
         return latest_release, source_uri
 
 
+class DISTAlertsSync(Sync):
+    """
+    Defines jobs to create new DIST alerts assets once a new release is available.
+    """
+
+    dataset = "umd_dist_alerts"
+    source_bucket = "earthenginepartners-hansen"
+    source_prefix = "DIST-ALERT"
+    input_calc = """
+        np.where((A>=30) & (A<255) & (B>0) & (C>=2) & (C<255),
+            np.where(C<4, 20000 + B, 30000 + B),
+            0        
```

Comment:
Change 0 to -1 here.  This is the recent request by Solomon that we mark no disturbance as -1 (no data) rather than zero, so alerts don't fade out improperly.